loopd/daemon.go

```diff
@@ -271,6 +287,12 @@ func (d *Daemon) initializeClient() error {
 
 // stop tries to gracefully shut down the daemon.
 func (d *Daemon) stop() error {
+	// Trying to stop the daemon multiple times isn't disallowed, we just
+	// need to make sure the logic itself is only run once.
```

Comment:
Is this expected to happen? Maybe we can at least log something, to possibly catch a bug around this. Same for `Run`

Reply:
Yes, explained the reason more clearly and also moved this to the commit where it makes more sense.